menuinst/win32.py

```diff
@@ -251,26 +254,28 @@ def remove(self):
 
     def create(self, remove=False):
         args = []
+        # cmd is our root installation interpreter
+        cmd = join(unicode_prefix, u"pythonw.exe").replace("\\", "/")
+        # each of these roll the subprocess cmd into args
         if "pywscript" in self.shortcut:
-            cmd = join(self.menu.prefix, u"pythonw.exe").replace("\\", "/")
+            subprocess_cmd = join(self.menu.prefix, u"pythonw.exe").replace("\\", "/")
             args = self.shortcut["pywscript"].split()
-            args = get_python_args_for_subprocess(self.menu.prefix, args, cmd)
+            args = get_python_args_for_subprocess(self.menu.prefix, args, subprocess_cmd)
         elif "pyscript" in self.shortcut:
-            cmd = join(self.menu.prefix, u"python.exe").replace("\\", "/")
+            subprocess_cmd = join(self.menu.prefix, u"python.exe").replace("\\", "/")
```

Comment:
This is now a behavioral change in that previously `python.exe` instead of `pythonw.exe` called `cwp` -- though I don't if there are any implications in using `pythonw` instead..